requirements.txt

```diff
@@ -0,0 +1,12 @@
+astroid==2.4.2
+autopep8==1.5.3
```

Comment:
`autopep8` is a dependency that will auto-format your code according to the Python style standards whenever the file is saved. It's what modified the structure of `spy_terminal.py` in this Pull Request. Learn more about those style guidelines which are published in Python's [PEP 8](https://www.python.org/dev/peps/pep-0008/)